package.json

```diff
@@ -81,7 +81,7 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-select": "^3.1.0",
+    "react-select": "^5.2.1",
```

Comment:
Had to update react-select because the className prefix was not being applied on the MenuPortal on version 3.1.0  and because of that the menu was being reverted the stock react-select styles.